xtext-gradle-plugin/src/integTest/java/org/xtext/gradle/test/AbstractIntegrationTest.xtend

```diff
@@ -39,6 +40,26 @@ class AbstractIntegrationTest {
 		}
 	'''
 	
+	protected def CharSequence getXtendPluginSnippet() '''
+		apply plugin: 'org.xtext.xtend'
+		
+		dependencies {
+			compile 'org.eclipse.xtend:org.eclipse.xtend.lib:2.9.0-SNAPSHOT'
+		}
+	'''
+	
+	protected def File createXtendHelloWorld() {
```

Comment:
The Xtend specific stuff could also go into an `AbstractXtendIntegrationTest` that inherits from `AbstractIntegrationTest` ;-)